### PR TITLE
Updated fields, jobTypes and technologies constants

### LINKED ISSUE #218: Update constants
- This PR contains a lot of values for fields and technologies but they should be revised and updated 
    -  #71 
- Should we update the value of max concurrent offers, regarding the feedback of companies and to maximize the quantity of offers to students
- Revise other constants such as number of maximum contacts

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #218 

We still need to update these in FE, since we haven't made the transition to nijobs-commons

### CODE REVIEW

File: src/models/constants/JobTypes.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const JobTypes = Object.freeze([
     "PART-TIME",
     "SUMMER INTERNSHIP",
     "CURRICULAR INTERNSHIP",
+    "GRANT",
```
Comment (DoStini) [inline src/models/constants/JobTypes.js:6]: Maube this should be Research grant. Are there any other ones that im missing ?

File: src/models/constants/TechnologyTypes.js [hunk 2]
```diff
@@ -1,22 +1,152 @@
...
+    "VirtualBox",
+    "Visual Basic",
+    "VMWare",
+    "VueJS",
+    "WebdriverIO",
+    "Windows",
+    "WordPress",
+    "Xamarin",
+    "Yii",
+    "Other"
```
Comment (DoStini) [inline src/models/constants/TechnologyTypes.js:149]: Add Solidity, Vyper, Simplicity, Rholang

**Review by DoStini — CHANGES_REQUESTED** [review]

File: src/models/constants/TechnologyTypes.js (hunk 2 above)
Comment (DoStini) [reply src/models/constants/TechnologyTypes.js:149]: Ruby on rails

Comment (DoStini) [reply src/models/constants/TechnologyTypes.js:149]: Gastsby

File: src/models/constants/JobTypes.js (hunk 1 above)
Comment (BrunoRosendo) [reply src/models/constants/JobTypes.js:6]: I don't think so

File: src/models/constants/TechnologyTypes.js (hunk 2 above)
Comment (BrunoRosendo) [reply src/models/constants/TechnologyTypes.js:149]: Gatsby is already on the list. I added the others

**Review by DoStini — APPROVED** [review]

**Pull request merged**